<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MyMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Files Encryption</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QTableView" name="fileListViewCrypto">
             <property name="focusPolicy">
              <enum>Qt::FocusPolicy::ClickFocus</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="CheckDeleteFiles">
            <property name="text">
             <string>Delete original item after encryption/decryption</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushEncrypt">
            <property name="text">
             <string>Encrypt</string>
            </property>
            <property name="icon">
             <iconset resource="rsc.qrc">
              <normaloff>:/pixmaps/encrypt.png</normaloff>:/pixmaps/encrypt.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushDecrypt">
            <property name="text">
             <string>Decrypt</string>
            </property>
            <property name="icon">
             <iconset resource="rsc.qrc">
              <normaloff>:/pixmaps/decrypt.png</normaloff>:/pixmaps/decrypt.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>CryptoPad</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPlainTextEdit" name="cryptoPadEditor">
          <property name="font">
           <font>
            <family>Monospace</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushEncryptTxt">
            <property name="text">
             <string>Encrypt</string>
            </property>
            <property name="icon">
             <iconset resource="rsc.qrc">
              <normaloff>:/pixmaps/encrypt.png</normaloff>:/pixmaps/encrypt.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushDecryptTxt">
            <property name="text">
             <string>Decrypt</string>
            </property>
            <property name="icon">
             <iconset resource="rsc.qrc">
              <normaloff>:/pixmaps/decrypt.png</normaloff>:/pixmaps/decrypt.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Logs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="textLogs"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="password_label">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="password_0">
        <property name="text">
         <string notr="true">mypassword</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="password_1">
        <property name="text">
         <string notr="true">mypassword</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Confirm:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushGenerator">
        <property name="text">
         <string>Generator</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/pixmaps/alpha-g-circle-outline.svg</normaloff>:/pixmaps/alpha-g-circle-outline.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkViewpass">
        <property name="text">
         <string>View Pass</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Files Encryption</string>
    </property>
    <addaction name="menuAddFiles"/>
    <addaction name="menuClearList"/>
    <addaction name="menuEncryptList"/>
    <addaction name="menuDecryptList"/>
    <addaction name="menuAbortJob"/>
    <addaction name="menuQuit"/>
    <addaction name="menuAddFolder"/>
   </widget>
   <widget class="QMenu" name="menuLog">
    <property name="title">
     <string>Logs</string>
    </property>
    <addaction name="menuSaveLogs"/>
    <addaction name="menuSelectAll"/>
    <addaction name="menuClearLogView"/>
    <addaction name="menuQuit3"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionThemeDark"/>
     <addaction name="actionThemeLight"/>
     <addaction name="actionThemeClassic"/>
    </widget>
    <addaction name="menuConfiguration"/>
    <addaction name="menuViewToolbar"/>
    <addaction name="menuTheme"/>
   </widget>
   <widget class="QMenu" name="menuPassword_Generator">
    <property name="title">
     <string>Utils</string>
    </property>
    <addaction name="menuGenerator"/>
    <addaction name="menuHashCalculator"/>
    <addaction name="menuArgon2Tests"/>
   </widget>
   <widget class="QMenu" name="menuInfos">
    <property name="title">
     <string>Infos</string>
    </property>
    <addaction name="menuAboutArsenic"/>
    <addaction name="menuAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuCryptopad">
    <property name="title">
     <string>Cryptopad</string>
    </property>
    <addaction name="menuOpenTxt"/>
    <addaction name="menuSaveTxt"/>
    <addaction name="menuSaveTxtAs"/>
    <addaction name="menuClearEditor"/>
    <addaction name="separator"/>
    <addaction name="menuEncryptTxt"/>
    <addaction name="menuDecryptTxt"/>
    <addaction name="menuQuit2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCryptopad"/>
   <addaction name="menuLog"/>
   <addaction name="menuConfig"/>
   <addaction name="menuPassword_Generator"/>
   <addaction name="menuInfos"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="menuAddFiles"/>
   <addaction name="menuClearList"/>
   <addaction name="separator"/>
   <addaction name="menuOpenTxt"/>
   <addaction name="menuSaveTxt"/>
   <addaction name="menuClearEditor"/>
   <addaction name="separator"/>
   <addaction name="menuGenerator"/>
   <addaction name="menuHashCalculator"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuAboutArsenic"/>
   <addaction name="menuQuit"/>
  </widget>
  <action name="menuAddFiles">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/pixmaps/add_file.png</normaloff>:/pixmaps/add_file.png</iconset>
   </property>
   <property name="text">
    <string>Add File(s)</string>
   </property>
  </action>
  <action name="menuQuit">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/pixmaps/close-outline.svg</normaloff>:/pixmaps/close-outline.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="menuSaveLogs">
   <property name="text">
    <string>Save as txt file</string>
   </property>
  </action>
  <action name="menuSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="menuConfiguration">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/pixmaps/settings-outline.svg</normaloff>:/pixmaps/settings-outline.svg</iconset>
   </property>
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="menuGenerator">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/pixmaps/alpha-g-circle-outline.svg</normaloff>:/pixmaps/alpha-g-circle-outline.svg</iconset>
   </property>
   <property name="text">
    <string>Pass Generator</string>
   </property>
  </action>
  <action name="menuHashCalculator">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/pixmaps/alpha-h-circle-outline.svg</normaloff>:/pixmaps/alpha-h-circle-outline.svg</iconset>
   </property>
   <property name="text">
    <string>Hash Calculator</string>
   </property>
  </action>
  <action name="menuAboutArsenic">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/pixmaps/comment-question-outline.svg</normaloff>:/pixmaps/comment-question-outline.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="menuAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="menuArgon2Tests">
   <property name="text">
    <string>Argon2 tests</string>
   </property>
  </action>
  <action name="menuViewToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Button Toolbar</string>
   </property>
  </action>
  <action name="menuOpenTxt">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/pixmaps/open_txt.png</normaloff>:/pixmaps/open_txt.png</iconset>
   </property>
   <property name="text">
    <string>Open Txt File</string>
   </property>
  </action>
  <action name="menuSaveTxt">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/pixmaps/save_txt.png</normaloff>:/pixmaps/save_txt.png</iconset>
   </property>
   <property name="text">
    <string>Save Txt File</string>
   </property>
  </action>
  <action name="menuEncryptTxt">
   <property name="text">
    <string>Encrypt Editor</string>
   </property>
  </action>
  <action name="menuDecryptTxt">
   <property name="text">
    <string>Decrypt Editor</string>
   </property>
  </action>
  <action name="menuSaveTxtAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="menuClearLogView">
   <property name="text">
    <string>Clear Log View</string>
   </property>
  </action>
  <action name="menuQuit2">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="menuQuit3">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="menuClearEditor">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/pixmaps/clear_txt.png</normaloff>:/pixmaps/clear_txt.png</iconset>
   </property>
   <property name="text">
    <string>Clear Editor</string>
   </property>
  </action>
  <action name="menuEncryptList">
   <property name="text">
    <string>Encrypt List</string>
   </property>
  </action>
  <action name="menuDecryptList">
   <property name="text">
    <string>Decrypt List</string>
   </property>
  </action>
  <action name="menuAbortJob">
   <property name="text">
    <string>Abort Job</string>
   </property>
  </action>
  <action name="menuClearList">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/pixmaps/remove_file.png</normaloff>:/pixmaps/remove_file.png</iconset>
   </property>
   <property name="text">
    <string>Remove All Files From List</string>
   </property>
  </action>
  <action name="menuAddFolder">
   <property name="text">
    <string>Add Folder</string>
   </property>
  </action>
  <action name="actionThemeDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionThemeLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionThemeClassic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Classic (platform-nativ)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushEncrypt</tabstop>
  <tabstop>pushDecrypt</tabstop>
  <tabstop>password_0</tabstop>
  <tabstop>password_1</tabstop>
 </tabstops>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
